lib/dor/services/indexing_service.rb

```diff
@@ -55,9 +57,11 @@ def self.reindex_pid(pid, *args)
         should_raise_errors = options.fetch(:raise_errors, true)
       end
 
-      obj = Dor.load_instance pid
-      solr_doc = reindex_object obj, options
-      index_logger.info "updated index for #{pid}"
+      obj = nil
+      load_stats = Benchmark.measure('load_instance') { obj = Dor.load_instance pid }
+      solr_doc = nil
+      to_solr_stats = Benchmark.measure('to_solr') { solr_doc = reindex_object obj, options }
+      index_logger.info "successfully updated index for #{pid} (metrics: #{load_stats.format('%n %rs %ts')}; #{to_solr_stats.format('%n %rs %ts')}; total #{load_stats.total + to_solr_stats.total}s)"
```

Comment:
i think it'd be nice to include a short comment inline about what's being written out in the `format` statement, and what the `total` indicates (i assume total runtime for the benchmarked operation?).